### PR TITLE
auto-build Bikeshed locally + update HTML on GH Pages (issue #18)

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

**Comment by cvan** [comment]
friendly ping (merge conflicts aside)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+node_js:
+  - "6"
+
+cache:
+  directories:
+    - node_modules
+
+script:
+  - npm run test
```
Comment (toji) [inline .travis.yml]: Unless this is a built in magic script target I don't see "test" defined anywhere in here.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "webvr",
+  "description": "WebVR Specification",
+  "private": true,
+  "version": "1.2.0",
```
Comment (toji) [inline package.json]: Let's update this to "2.0.0" just to avoid confusion.

**Comment by toji** [comment]
Left a couple of comments and we should get rid of the index conflict, but otherwise this looks fine. Feel free to merge it once those are addressed.

File: package.json (hunk 2 above)
Comment (cvan) [inline package.json]: sure

**Comment by cvan** [comment]
@toji: tested it some more, plus the local-development flow. made some changes; feel free to re-review this. thanks!

**Comment by toji** [comment]
LGTM, thanks!

**Comment by cvan** [comment]
Odd, this doesn't seem to pushing commits containing the generated Bikeshed. The [builds pass now though](https://travis-ci.org/w3c/webvr/builds/221561262), and the commands exited without error, so perhaps Travis is silently failing on my GitHub commit-author token.

I have a couple ideas to investigate. I'll make a follow-up PR soon.

**Pull request merged**